.circleci/config.yml

```
@@ -33,6 +41,30 @@ jobs:
       - run: find . -name *.pyc -delete
       - run: tox -e py27
 
+  test-environment:
+    machine:
+      image: default
```

As it appears you noticed, the default image is from 2017, it appears. This ubuntu image, according to https://circleci.com/docs/2.0/configuration-reference/#available-machine-images appears to be the most recent option
```suggestion
      image: ubuntu-1604:201903-01
      docker_layer_caching: true
```